azure/templates/marketplace-stack-management/mainTemplate.json

```diff
@@ -13,9 +13,15 @@
       "type": "string",
       "allowedValues": [
         "R80.30 - Bring Your Own License",
-        "R80.30 - Pay As You Go (MGMT25)"
+        "R80.30 - Pay As You Go (MGMT25)",
+        "R80.40 - Bring Your Own License",
+        "R80.40 - Pay As You Go (MGMT25)",
+        "R81 - Bring Your Own License",
+        "R81 - Pay As You Go (MGMT25)",
+        "R81.10 - Bring Your Own License",
+        "R81.10 - Pay As You Go (MGMT25)"
       ],
-      "defaultValue": "R80.30 - Bring Your Own License",
+      "defaultValue": "R80.40 - Bring Your Own License",
```

Comment:
Why it is R80.40 and not R81.10?

Reply:
Update default to "R81.10 - Bring Your Own License"